Harvest.Web/ClientApp/src/Projects/InvoiceListContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { Invoice } from "../types";
+import { InvoiceTable } from "./InvoiceTable";
+
+interface Props {
+    projectId: any;
+}
+
+
+export const InvoiceListContainer = (props: Props) => {
+    const [invoices, setInvoices] = useState<Invoice[]>([]);
+
+    useEffect(() => {
+    // get rates so we can load up all expense types and info
+        const cb = async () => {
+            const response = await fetch(`/Project/Invoices/${props.projectId}`);
+
+      if (response.ok) {
+        setInvoices(await response.json());
+      }
+    };
+
+    cb();
+  }, []);
```

Comment:
you should put `props.projectId` here since if project Id changes you would want to do the query again